api/services/course.js

```diff
@@ -16,6 +17,18 @@ const getCoursePreview = async id => {
   return course;
 };
 
+const getCoursesForCarousel = async id => {
+  const course = await db.Courses.findAll({
+    attributes: ['id', 'title', 'description', 'numberOfLessons'],
+    order: Op.random(),
+    limit: 10,
+  });
+  if (!course) throw new errors.NotFoundError('Courses not found');
+
+  return course;
+};
```

Comment:
Might If DB server is unavailable the error doesn't process and it will not send to the client message about the error.
[https://trello.com/c/eWnlic6V/190-task-104-getting-courses-for-carousel](url)